addons/binding/org.openhab.binding.netatmo/ESH-INF/thing/welcomeEvent.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+      
+    <thing-type id="NAWelcomeEvent">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="netatmoapi" />
+        </supported-bridge-type-refs>
+        
+        <label>Welcome Camera Event</label>
+        <description>This represents a event at home</description>
+
+        <channels>
+            <channel id="welcomeEventId" typeId="id"></channel>
+        	<channel id="welcomeEventType" typeId="type"></channel>
+            <channel id="welcomeEventTime" typeId="time"></channel>
+           	<channel id="welcomeEventCameraId" typeId="camera_id"></channel>
+        	<channel id="welcomeEventPersonId" typeId="person_id"></channel>
+        	<channel id="welcomeEventSnapshotId" typeId="snapshot_id"></channel>
+            <channel id="welcomeEventSnapshotVersion" typeId="snapshot_version"></channel>
+            <channel id="welcomeEventSnapshotKey" typeId="snapshot_key"></channel>
+            <channel id="welcomeEventVideoID" typeId="video_id"></channel>
+            <channel id="welcomeEventVideoStatus" typeId="video_status"></channel>
+            <channel id="welcomeEventIsArrival" typeId="is_arrival"></channel>
+            <channel id="welcomeEventMessage" typeId="message"></channel>
+            <channel id="welcomeEventSubType" typeId="sub_type"></channel>
+            <channel id="welcomeEventPictureUrl" typeId="event_picture_url"></channel>
+            <channel id="welcomeEventVideoPoorUrl" typeId="event_video_poor_url"></channel>                                   
+            <channel id="welcomeEventVideoLowUrl" typeId="event_video_low_url"></channel>                                   
+            <channel id="welcomeEventVideoMediumUrl" typeId="event_video_medium_url"></channel>                                   
+            <channel id="welcomeEventVideoHighUrl" typeId="event_video_high_url"></channel>    
+                                                     
+        </channels>
+              
+        <properties>
+             <property name="actualApp">app_welcomecamera</property>
+        </properties>         
+        
+        <config-description>
+            <parameter name="welcomeHomeId" type="text" required="true" readOnly="true">
+            	<label>Home ID</label>
+            	<description>Id of the home</description>
+            </parameter>
+
+            <parameter name="welcomeEventId" type="text" required="true" readOnly="true">
```

Comment:
I get the welcomeEventId from the netatmo api, and the events are dynamic and they change, maybe i can remove the eventId parameter, and just leave the channel?